lispusers/GITFNS

```diff
@@ -16,7 +16,7 @@
         (* ;; "Set up")
 
         (FILES (SYSLOAD FROM LISPUSERS)
```

Comment:
I've taken to changing these to just (FILES file1 file2 ..) rather than (FILES (SYSLOAD FROM wherever) fiile1 file2 ..) with the idea that LDFLG is inherited and DIRECTORIES should be set up. Let's discuss.s